charts/prometheus-redis-exporter/Chart.yaml

```
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: v1.54.0
 description: Prometheus exporter for Redis metrics
 name: prometheus-redis-exporter
-version: 5.6.0
+version: 5.6.1
```

Removing field `redisAddressConfig.configmap` makes the chart incompatible for the users making use of the field - an upgrade will fail without a prior adjustment. In this respect, the change is major deserving an upgrade note in README.

```suggestion
version: 6.0.0
```